README.md

```diff
@@ -1 +1 @@
-# Push-2-Tweet
+#<a id="section0"></a> Push-2-Tweet
```

Comment:
Add just a brief descriptsion (I don't think a full documentation is needed, but it seems really strange to have it blank).